src/js/base/module/Editor.js

```diff
@@ -190,6 +190,7 @@ export default class Editor {
       let linkUrl = linkInfo.url;
       const linkText = linkInfo.text;
       const isNewWindow = linkInfo.isNewWindow;
+      const checkProtocol = linkInfo.checkProtocol;
```

Comment:
`linkInfo` does not contain `checkProtocol` property.

Reply:
To put `checkProtocol` into `linkInfo`, please add the information in `getLinkInfo` function of `Editor.js`.